var hello = function() {
  console.log("Inside unit-tests!!!")
  throw Exception('hej')
}();
